src/Map.jsx

```diff
@@ -185,7 +186,9 @@ class Map extends Component {
       .reduce((layers, route) => [
         ...layers,
         this.state.showStops
-          ? getStopMarkersLayer(route, marker => this.getStopInfo(route, marker.object.position))
+        // getStopMarkersLayer(route,
+        // marker => this.getStopInfo(route, marker.object.position), this.state.selectedStops)
```

Comment:
@EddyIonescu annoying 100 char limit issue again. Wanted to pass in marker.object.position but no matter how I tried to break up the function call, ESLint got mad haha
Maybe you can figure it out?

Reply:
```
  this.getStopInfo(
    route,
    marker.object.position,
  )
```